README.md

```diff
@@ -52,11 +52,12 @@ First-timers is built with [Probot](https://probot.github.io/).
 
 Thank you to everyone who has helped with this project.
 <!-- Contributors START
-Michael_McCombie michaelmccombie https://twitter.com/michaelbuilds design
-Contributors END -->
+ Michael_McCombie michaelmccombie https://twitter.com/michaelbuilds design
+ Gregor_Martynus gr2m https://twitter.com/gr2m mentor
+ Contributors END -->
 <!-- Contributors table START -->
-| <img src="https://avatars.githubusercontent.com/michaelmccombie?s=100" width="100" alt="Michael McCombie" /><br />[<sub>Michael McCombie</sub>](https://twitter.com/michaelbuilds)<br />[🎨](https://raw.githubusercontent.com/hoodiehq/first-timers-bot/51742c62ae3e4e2be7e58d170a9eab73a3871bf4/assets/avatar.png)|
-| :---: |
+| <img src="https://avatars.githubusercontent.com/michaelmccombie?s=100" width="100" alt="Michael McCombie" /><br />[<sub>Michael McCombie</sub>](https://twitter.com/michaelbuilds)<br />🎨 | <img src="https://avatars.githubusercontent.com/gr2m?s=100" width="100" alt="Gregor Martynus" /><br />[<sub>Gregor Martynus</sub>](https://twitter.com/gr2m)<br />👨🏻‍🏫 |
```

Comment:
it removed the link to the avatar that Micheal created

Reply:
Good catch! Editing it now